src/main/java/gift/exception/ProductErrorCode.java

```diff
@@ -0,0 +1,15 @@
+package gift.exception;
+
+public enum ProductErrorCode {
+    HAS_KAKAO_WORD("\"kakao\"가 포함된 문구는 담당자와 협의 후 사용할 수 있습니다.");
+
+    private final String message;
+
+    private ProductErrorCode(String message) {
```

Comment:
enum의 생성자는 기본으로 private이고 접근제어자를 변경할 수 없기에 생략하는 편입니다.
```suggestion
   ProductErrorCode(String message) {
```